src/Console/ModelsCommand.php

```diff
@@ -708,11 +722,8 @@ protected function setProperty($name, $type = null, $read = null, $write = null,
             $this->properties[$name]['comment'] = (string) $comment;
         }
         if ($type !== null) {
-            $newType = $this->getTypeOverride($type);
-            if ($nullable) {
-                $newType .= '|null';
-            }
-            $this->properties[$name]['type'] = $newType;
+            $this->properties[$name]['type'] =
+                $this->getTruePropertyType($type, $nullable);
```

Comment:
I wouldn't mind this being one line, 80 chars is great but I think up until 100 chars we're good

Reply:
I'll update it.